Rakefile

```
@@ -3,15 +3,15 @@
 begin
   require 'rspec/core/rake_task'
   RSpec::Core::RakeTask.new(:spec)
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
 end
 
 begin
   require 'rubocop/rake_task'
   RuboCop::RakeTask.new(:rubocop) do |task|
     task.fail_on_error = false
   end
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
```

Do not suppress exceptions.